iml-api/src/graphql.rs

```diff
@@ -80,11 +80,35 @@ struct TargetResource {
     cluster_hosts: Vec<i32>,
 }
 
+struct BannedTargetResource {
+    resource: String,
+    cluster_id: i32,
+    host_id: i32,
+    mount_point: Option<String>,
+}
+
+#[derive(juniper::GraphQLObject)]
+/// A Corosync banned resource
+struct BannedResource {
+    /// The resource id
+    id: String,
+    /// The id of the cluster in which the resource lives
+    cluster_id: i32,
+    /// The resource name
+    resource: String,
+    /// The node in which the resource lives
+    node: String,
+    /// The assigned weight of the resource
+    weight: i32,
+    /// Is master only
+    master_only: bool,
+}
+
 pub(crate) struct QueryRoot;
 pub(crate) struct MutationRoot;
 
 #[derive(GraphQLEnum)]
-enum SortDir {
+pub enum SortDir {
```

Comment:
I can't find where it's used outside this module. Why does it need to be public?